atlas/commands/bootstrap/expand.go

```diff
@@ -0,0 +1,232 @@
+package bootstrap
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/jinzhu/inflection"
+	"io"
+	"log"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/iancoleman/strcase"
+)
+
+type finalTemplate struct {
+	AppName string
+	WithDatabase bool
+	R []templateResource
+}
+
+type templateResource struct {
+	NameCamel string
+	NameCamels string
+	NameLowerCamel string
+	NameLowerCamels string
+	NameSnake string
+	NameSnakes string
+	MigrateVer string
+	Path string
+	WithDatabase bool
+}
+
+func expandResource(appName, expandName string, withDatabase bool) error {
+
+	lines, err := readLines("../" + expandName)
+	if err != nil {
+		panic(err)
+	}
+
+	r := []templateResource{}
+
+	migrateNum := 2
+	migrateStr := ""
+
+	for _, name := range lines {
+		if (migrateNum > 9) {
+			migrateStr = "000" + strconv.Itoa(migrateNum)
+		} else {
+			migrateStr = "0000" + strconv.Itoa(migrateNum)
+		}
```

Comment:
`migrateStr = fmt.Sprintf("%05d", migrateNum)`
Is a more concise syntax, and handles the unlikely case of n >= 98 migration files to generate